packages/hint/src/lib/utils/schema-validator.ts

```diff
@@ -122,6 +208,64 @@ const prettify = (errors: ajv.ErrorObject[]) => {
     return result;
 };
 
+const groupMessages = (errors: ISchemaValidationError[]): GroupedError[] => {
```

Comment:
There's a lot going on here. May be worth a comment explaining what the routine does (criteria for grouping, an example, etc.).